.travis.yml

```diff
@@ -37,6 +40,8 @@ matrix:
       env: COVERAGE=true TEST=false
     - node_js: "lts/*"
       env: KARMA=true TEST=false
+    - node_js: "lts/*"
+      env: HAPPO=true TEST=false
```

Comment:
do we want this to be an allowed failure, or will it always robustly complete?

Reply:
I added the name of the variable in .travis.yml, then used the commit message to explain things. Anyone ending up git-blaming these should land in this commit: 7c4e388.